### PR TITLE
Ran CNN notebook through 10 epochs

### PR DESCRIPTION
**Pull request by chantsin**
CNN seems to be a little overfit:
training ~ 99.97
validation ~ 98.75

### CODE REVIEW

**Comment by chantsin** [comment]
Just added dropout layer, currently set to 30% but haven't ran it yet. Can play around with it for better accuracy.

**Review by clem21chan — APPROVED** [review]
looks good

**Review by tilovashahrin — COMMENTED** [review]
looks good!
